articles/azure-monitor/platform/alerts-log.md

```diff
@@ -257,6 +280,9 @@ New-AzResourceGroupDeployment -Name AlertDeployment -ResourceGroupName ResourceG
 > [!NOTE]
 > Azure CLI support is only available for the scheduledQueryRules API version `2020-05-01-preview` and above. Pervious API version can use the Azure Resource Manager CLI with templates as described below. If you use the legacy [Log Analytics Alert API](api-alerts.md), you will need to switch to use CLI. [Learn more about switching](./alerts-log-api-switch.md).
 
+> [!IMPORTANT]
+> When using cross-resource query in log alert, the usage of [authorizedResources](/rest/api/monitor/scheduledqueryrules/createorupdate#source) is mandatory and user must have access to the list of resources stated
+
```

Comment:
CLI is only for the new API version that doesn't require authorizedResources anymore.
Other than that looks good, so approved once this is removed from the PR.